pandas/core/frame.py

```diff
@@ -7458,23 +7458,24 @@ def isin(self, values):
         1  False  False
```

Comment:
Can you start the example before the "when values is a list" with the constructor, and displaying the content of the original Dataframe (i.e. `>>> df`)

Also, I think the docstring would be clearer if we use a more real world example. For example, if the index is an animal name, and the columns are `num_legs` and `num_wings`, a `df.isin([2])` having a `True` in `falcon/wings` is very easy to see what it's doing. And not so much in a `foo`, `A`, `1, 2, 3` kind of example.